### PR TITLE
README for a clean install

### PR DESCRIPTION
**Pull request by jaylingelbach**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,101 @@
 # SpatialConnect
+
+#### Fresh install of EFC and the associated dependencies. ####
+
+#### **Dependencies List** ####
+What you will need to have installed prior to running the program:
+
+##### Git repositories #####
+* [spatialconnect-mobile](https://github.com/boundlessgeo/spatialconnect-mobile)
+* [React-native-maps](https://github.com/boundlessgeo/react-native-maps)
```
Comment (mrcnc) [inline README.md]: Can you keep these lower case to match the repo names?

File: README.md [hunk 2]
```diff
@@ -1 +1,101 @@
...
+
+###### iOS specific ######
+* Xcode
+* Carthage
+* Fabric
+* Crashlytics.framework
+
+###### Android specific ######
+* Android Studio
+* Java SE Runtime 1.7
```
Comment (mrcnc) [inline README.md]: You can remove this line.  A JDK includes a JRE so it makes sense to say "install the jdk" but not install both a JDK and a JRE

File: README.md [hunk 3]
```diff
@@ -1 +1,101 @@
...
+``
+npm i
+``.
+* In the _spatialconnect-mobile folder_
+``
+npm i
+``.
+* Next
+``
+npm i react-native-spatialconnect
```
Comment (mrcnc) [inline README.md]: This command shouldn't be needed if you previously ran `npm i`, which installs all the dependencies, including react-native-spatialconnect.

File: README.md [hunk 4]
```diff
@@ -1 +1,91 @@
...
+**or** in the _ios_ folder of _spatialconnect-mobile_ open the SCMobile.xcodeproj file.
+
+#### Android setup ####
+* Install [Android Studio](https://developer.android.com/studio/index.html)
+* Add the gradle.properties file to _spatialconnect-mobile_ > _android_ folder.
+* Next
+``
+npm i
+``.
+* In the _spatialconnect-mobile folder_
```
Comment (mrcnc) [inline README.md]: Is this a duplicate step?  I see `npm i` 2 times?

File: README.md [hunk 5]
```diff
@@ -1 +1,91 @@
-# SpatialConnect
+
+#### Fresh install of EFC and the associated dependencies. ####
+
+#### **Dependencies List** ####
+What you will need to have installed prior to running the program:
+
+##### Git repositories #####
+* [spatialconnect-mobile](https://github.com/boundlessgeo/spatialconnect-mobile)
```
Comment (mrcnc) [inline README.md]: Maybe add the clone commands explicitly here so developers can copy & paste?

**Comment by mrcnc** [comment]
@jaylingelbach can you also update your ticks to use 3 ticks to follow the [Github Flavored Markdown](https://help.github.com/articles/creating-and-highlighting-code-blocks/) syntax?

File: README.md (hunk 4 above)
Comment (jaylingelbach) [reply README.md]: Yes. Thank you. :)

**Comment by mrcnc** [comment]
The markdown is still messed up as you can see [here](https://github.com/jaylingelbach/SpatialConnect/blob/4ab66d580af56f39c6b112a17dd971a9e60aa11b/README.md).  Also, put a blank line before and after your triple backticks so it's easier to read.

**Comment by jaylingelbach** [comment]
Oh. Weird. It looked fine in my editor. First time using markup.
screen shot 2017-04-04 at 7 29 57 pm

**Comment by jaylingelbach** [comment]
Is this better? Sorry about all the back and fourth on this one.

**Comment by mrcnc** [comment]
Back and forth is a good thing!  Learning how to write [markdown](https://guides.github.com/features/mastering-markdown/) is a useful skill so I want to make sure you understand what's going on.  I've found that the markdown preview in text editors (even Atom) doesn't always match what you'll see on GitHub so I usually push code and view it directly from GitHub. 

One thing I noticed is that you seem to be using the markdown headers incorrectly.  For instance `#### Some heading ####` when you only need `#### Some heading`.  Also, within a heading, there is no need to make something **bold** because the header will already do that for you.  So instead of `#### **Dependencies List** ####`, you can use `#### Dependencies List`

Finally, it looks like you are trying to make a list of `<dependency name>: <command to install>`.  If that is the case, then you don't want to use the triple backticks, you'll want to use the single backticks like this:

```
* [spatialconnect-mobile](https://github.com/boundlessgeo/spatialconnect-mobile) `git clone https://github.com/boundlessgeo/spatialconnect-mobile`

* [react-native-maps](https://github.com/boundlessgeo/react-native-maps) `git clone https://github.com/boundlessgeo/react-native-maps`

* [react-native-spatialconnect](https://github.com/boundlessgeo/react-native-spatialconnect) `
git clone https://github.com/boundlessgeo/react-native-spatialconnect`
```

which looks like this when it's rendered:

* [spatialconnect-mobile](https://github.com/boundlessgeo/spatialconnect-mobile) `git clone https://github.com/boundlessgeo/spatialconnect-mobile`

* [react-native-maps](https://github.com/boundlessgeo/react-native-maps) `git clone https://github.com/boundlessgeo/react-native-maps`

* [react-native-spatialconnect](https://github.com/boundlessgeo/react-native-spatialconnect) `
git clone https://github.com/boundlessgeo/react-native-spatialconnect`

File: README.md [hunk 6]
```diff
@@ -1 +1,104 @@
...
+`
+git clone https://github.com/boundlessgeo/spatialconnect-mobile
+`
+
+* [react-native-maps](https://github.com/boundlessgeo/react-native-maps)
+`
+git clone https://github.com/boundlessgeo/react-native-maps
+`
+
+* [react-native-spatialconnect](https://github.com/boundlessgeo/react-native-spatialconnect)
```
Comment (frankrowe) [inline README.md]: this is on npm, so no need to download from github

File: README.md [hunk 7]
```diff
@@ -1 +1,104 @@
...
+#### Dependencies List
+What you will need to have installed prior to running the program:
+
+##### Git repositories
+* [spatialconnect-mobile](https://github.com/boundlessgeo/spatialconnect-mobile)
+`
+git clone https://github.com/boundlessgeo/spatialconnect-mobile
+`
+
+* [react-native-maps](https://github.com/boundlessgeo/react-native-maps)
```
Comment (frankrowe) [inline README.md]: currently we are using the airbnb npm version of react-native-maps, so this is not needed for now

**Comment by mrcnc** [comment]
👍

**Pull request merged**